src/CoreWCF.ConfigurationManager/src/CoreWCF/Configuration/ExtensionElement.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Configuration;
+
+namespace CoreWCF.Configuration
+{
+    internal class ExtensionElement : ConfigurationElement
+    {
+        string typeName;
+
```

Comment:
Missing private accessibility modifier, and naming convention is underscore for private fields.